tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/TestHelper.cs

```diff
@@ -362,6 +362,7 @@ public void EnableIast(bool enable = true)
         public void EnableRasp(bool enable = true)
         {
             SetEnvironmentVariable(ConfigurationKeys.AppSec.RaspEnabled, enable.ToString().ToLower());
+            SetEnvironmentVariable(ConfigurationKeys.AppSec.StackTraceEnabled, "true");
```

Comment:
I thought you said `StackTraceEnabled=true` is the default? 🤔 Is this necessary?

Reply:
Not really. I will remove it. Thanks!